### DISCUSSION TITLE
GitHub Desktop: Partial Stash

### DISCUSSION THREAD

**Discussion by marlonjames71** [community/community#47095] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is there a way to only stash selected files? Currently I'm only seeing the ability to stash all files.

GitHub Desktop Version 3.1.6 (arm64)

Thanks!

**Comment by al1103** [comment]
no way you can just download them all

**Reply by marlonjames71** [reply]
Well that's a bummer

**Comment by steveward** [comment]
@marlonjames71 at this time you can only stash all files, but we're tracking this in https://github.com/desktop/desktop/issues/11531 if you'd like to give it a 👍🏻 .

**Reply by marlonjames71** [reply]
> @marlonjames71 at this time you can only stash all files, but we're tracking this in https://github.com/desktop/desktop/issues/11531 if you'd like to give it a 👍🏻 .

Upvoted and left comment. Thanks for the heads up.
